spec/unit/worker_spec.rb

```diff
@@ -68,7 +68,7 @@ class JobClass; end
         job.should respond_to(:fail).with(2).arguments
         job.should_receive(:fail) do |group, message|
           last_line = message.split("\n").last
-          expect(last_line).to match(/base\.rb:\d+:in `around_perform'/)
+          expect(last_line).to match(/base\.rb:\d+:in `perform'/)
```

Comment:
This test is a bit fragile, since it depends on whether call `around_perform` is the first thing in `perform` or not.